flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/calcite/FlinkRelBuilder.java

```diff
@@ -88,7 +89,8 @@ private FlinkRelBuilder(Context context, RelOptCluster cluster, RelOptSchema rel
         super(context, cluster, relOptSchema);
 
         this.toRelNodeConverter =
-                new QueryOperationConverter(this, context.unwrap(FlinkContext.class).isBatchMode());
+                new QueryOperationConverter(
+                        this, ShortcutUtils.unwrapContext(context).isBatchMode());
```

Comment:
nit: use static imports everywhere to make the code more readable

Reply:
you mean everywhere I use ShortcutUtils in this commit?